Mlem/App Constants.swift

```diff
@@ -11,22 +11,24 @@ import UIKit
 
 struct AppConstants
 {
+    static let cacheSize = 500_000_000 // 500MiB in bytes
+    static let urlCache: URLCache = URLCache(memoryCapacity: cacheSize, diskCapacity: cacheSize)
```

Comment:
500MB memory for images? seems a tad excessive.

Reply:
You might have a point 🤔 the maximum cache size of Apollo is 200 MB

And after scrolling my feed until my finger hurt, I only got to 148 MB of cache usage 

It's important to note though, that this cache is managed by iOS and will be cleared if the device is low on memory automatically (it will be cleared from disk If the device is running out of storage, or if the application is requesting too much memory, it’s also a reason for iOS to clear it from memory)

Plus, we have the button in the settings to clear it if the user wants to make more space for other stuff 

To me those two points justify having a larger cache (since it will be cleared without hesitation if it bothers the user at any point)

But I leave this decision in your capable hands 😉

Reply:
Apollo kept that cache size small with a mix of reddit doing a ton of heavy lifting with all manner of image optimization for almost every imaginable use case, and Apollo employing pretty aggressive cache and active memory management. images in your feed, for example, got flushed pretty quickly when you were scrolling, and only ones recently in your viewport remained when you tapped on a post. if you were gone from your feed long enough, they got dumped. even comments in a post didn't hang around for long and, when swiping pack to your feed and scrolling again, a lot would have to be reloaded.

I would run into this problem a lot when I would ride the subway and have no connection between stations.